### DISCUSSION TITLE
Improvement to Apex tests sidebar.

### DISCUSSION THREAD

**Discussion by mihailTr-19** [forcedotcom/salesforcedx-vscode#6272] [Ideas]
When I run all my Apex tests, and only one of them failed I have to scroll throught all apex tests to find the red mark.
But if I can collapse them to view Apex classes only I could find the class with the error, and after it open that class.

Add collapse all functionality to the Apex tests sidebar.
![image](https://user-images.githubusercontent.com/53862745/222355611-95bad5dd-86c2-409f-a652-5a47afdb0416.png)

Like we have it in the Explorer.
![image](https://user-images.githubusercontent.com/53862745/222355774-adf66139-e844-4dcf-a274-fb56693d2e3a.png)

**Comment by goto-dev-null** [comment]
💯 Without this the test pane is unusable. Even just defaulting to having every class collapsed instead of expanded would make it usable; I can't imagine anyone preferring to scroll through through 50x more lines to find 1 file.

A filter/jump-to feature would also make it much more usable IMO. Even with every test file collapsed, it's still a chore to find it in the list.

But I just wanna collapse them more than anything.
